### ISSUE TITLE
Linux build of protoc-gen-grpc-java does not support java lite

### ISSUE THREAD

**Issue by mythz** [grpc/grpc-java#6506]
### What version of gRPC-Java are you using?

Using v1.25.0 pre-built libraries from:
https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/1.25.0/

### What is your environment?

Windows 10 and Ubuntu 16.04

### What did you expect to see?

Expect the same version of the behavior of the **protoc-gen-grpc-java** plugin to generate "java lite" on both Windows and Linux.

On Windows we can generate [java lite](https://github.com/protocolbuffers/protobuf/blob/master/java/lite.md) gRPC clients with:

```
protoc.exe -I . -I "C:\path\to\include" --java_out=lite:out --plugin="C:/path/to/protoc-gen-grpc-java.exe" --grpc-java_out=lite:out services.proto
```

### What did you see instead?

But the Linux build of the same plugin version on Linux:

```
protoc -I . -I "/path/to/protoc/include" --java_out=lite:out --plugin="/path/to/protoc-gen-grpc-java" --grpc-java_out=lite:out services.proto` 
```

Fails with:

    command failed: --java_out: services.proto: Unknown generator option: lite

### Steps to reproduce the bug

Try running command above on Ubuntu 16.04 using the Linux plugin from https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/1.25.0/protoc-gen-grpc-java-1.25.0-linux-x86_64.exe (renamed to protoc-gen-grpc-java).

Is there an official or more up-to-date repository for accessing the pre-built java plugin then
https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/ ?

**Comment by ericgribkoff** [comment]
The linux command you gave works fine for me. The error you report:

```
command failed: --java_out: services.proto: Unknown generator option: lite
```

Indicates an unknown option for the `--java_out` flag, which is parsed by `protoc` itself, not the gRPC plugin. I would guess that the protoc installation on your windows machine supports proto lite, whereas your linux installation does not. I'm not aware of any specific need to "enable" protolite with recent versions of the protobuf library ([nothing like that is mentioned here](https://github.com/protocolbuffers/protobuf/blob/master/java/lite.md)), so you might want to try updating to a more recent protobuf on your linux machine (e.g., via the instructions [here](https://github.com/grpc/grpc-java/blob/master/COMPILING.md#build-protobuf)). If you continue to have issues with the `--java_out` parameter you should likely bring that to the attention of the protobuf folks at https://github.com/protocolbuffers/protobuf/.

I'm going to mark this closed since I don't see any bug with gRPC Java's behavior here. If I've missed something, please comment and I can reopen.

**Comment by mythz** [comment]
Ahh thx for pointing me to the right place, can confirm it now works after updating `protoc`.

Is there an official place to be able to get the latest pre-built binaries?

This only goes up to 1.25.0 but [grpc-java examples](https://github.com/grpc/grpc-java/blob/master/examples/build.gradle#L25) are referencing `1.27.0-SNAPSHOT`:
https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/

Is there a public build server I can pull the latest pre-built binaries from?

**Comment by ericgribkoff** [comment]
The code in master, including the examples, has been bumped to 1.27 since we are already past the 1.26 branch cut; 1.26 itself will be released one week from today, so currently 1.25 is still our most recent release. The Maven repos are the best place to get our released versions.
